app/src/main/java/com/skyyo/samples/features/textGradient/TextGradientScreen.kt

```diff
@@ -0,0 +1,72 @@
+package com.skyyo.samples.features.textGradient
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.skyyo.samples.features.textGradient.composables.RunningGradientText
+import com.skyyo.samples.features.textGradient.composables.ShimmerGradientText
+
+@Composable
+fun TextGradientScreen() {
```

Comment:
let's please do the same as we did before and show url which we used above the screen